### PR TITLE
fix(style): changed colors for dark-mode

### PR DESCRIPTION
**Pull request by AlyChloe**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes https://github.com/freeCodeCamp/freeCodeCamp/issues/16243

#### Description
Tweaked CSS background colors for dark mode

### CODE REVIEW

**Comment by AlyChloe** [comment]
@raisedadead thanks for the heads up! 

I removed my [previous](https://github.com/freeCodeCamp/freeCodeCamp/pull/16376) PR because I couldn't figure out how to squash merge commits? So I thought moving my changes to a whole new branch would be easier.

Let me know if I need to make any other changes..

**Comment by raisedadead** [comment]
LGTM. Thanks for the fresh PR @AlyChloe

File: common/app/Nav/nav.less [hunk 1]
```diff
@@ -153,6 +153,28 @@ li.nav-avatar {
...
+    ::-ms-placeholder {
+      color: @night-text-color;
+    }
+
+    ::placeholder {
+      color: @night-text-color;
+    }
+
+    .fcc_input {
+      background: @night-search-color;
```
Comment (tomasvn) [inline common/app/Nav/nav.less]: Here if you are adding just the color, then you should prefer longhand property for the background color.
If you set here a shorthand you will eventually set other background rules with their initial value. This is considered as an anti-pattern.

File: common/app/routes/Challenges/challenges.less [hunk 2]
```diff
@@ -212,6 +215,7 @@
 .@{ns}-preview {
   .max-element-height();
   width: 100%;
+  background: @silver-chalice;
```
Comment (tomasvn) [inline common/app/routes/Challenges/challenges.less]: Again prefer a longhand property for background color.

**Review by tomasvn — CHANGES_REQUESTED** [review]
Removing anti patterns in LESS file

File: common/app/Nav/nav.less (hunk 1 above)
Comment (AlyChloe) [reply common/app/Nav/nav.less]: @tomasvn can you explain to me what you mean by 'longhand property'? 

Thank you!

Comment (AlyChloe) [reply common/app/Nav/nav.less]: @tomasvn nvm :) let me know if changes look good to you!

**Review by raisedadead — APPROVED** [review]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @AlyChloe for this, works like a charm. And congrats on your first contribution to the code base, keep them coming.

Thanks @tomasvn for the QA feedback. Much appreciated.

Happy Contributing!

**Pull request merged**